### PR TITLE
Moar nondestructive tests

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

File: test/verify/check-system-info [hunk 1]
```diff
@@ -391,10 +391,12 @@ class TestSystemInfo(MachineCase):
         # restarts timesyncd to pick up the new config
         wait(lambda: get_timesyncd_start() > prev_timesyncd_start, delay=0.2)
 
+    @nondestructive
```
Comment (martinpitt) [inline test/verify/check-system-info:394]: commit message subject has an excess "to"

**Review by martinpitt — CHANGES_REQUESTED** [review]
Looks nice, thanks! However, please drop the no-test before the final run.

**Comment by jelly** [comment]
> Looks nice, thanks! However, please drop the no-test before the final run.

Ofcourse, I have some more WIP stuff locally, just wanted to run a single distro before running it on all as one distro should already shown some breakages.

File: test/verify/check-users-roles [hunk 2]
```diff
@@ -103,10 +103,12 @@ class TestRoles(MachineCase):
 
         self.allow_restart_journal_messages()
 
+    @nondestructive
     def testDynamic(self):
         m = self.machine
         b = self.browser
 
+        self.addCleanup(m.execute, "groupdel docker || true")
```
Comment (martinpitt) [inline test/verify/check-users-roles:112]: We shouldn't need this, testlib.py already restores passwd/group and so on. But of course it doesn't really hurt.

**Review by martinpitt — APPROVED** [review]
Nice, thanks!

**Pull request merged**